chcemvediet/templates/main/snippets/header/menu_left.html

```diff
@@ -25,7 +25,11 @@
 </a>
 
 <a class="chv-navbar-item {{ request|active:'inforequests'|yes:"active" }}"
-   href="{% url 'inforequests:mine' %}">
+   {% if user.is_authenticated %}
+     href="{% url 'inforequests:mine' %}">
+   {% else %}
+     href="{% url 'inforequests:index' %}">
```

Comment:
Uff, takto jeden otvaraci `<` a dva zatvaracie `>` je humus. Nie len ze je to tazko citatelne, ale este to aj rozbije syntax highligting, kedze tag je 2x uzavrety. Ked uz tak takto:

```
<a class="chv-navbar-item {{ request|active:'inforequests'|yes:"active" }}" 
   {% if user.is_authenticated %}
     href="{% url 'inforequests:mine' %}"
   {% else %}
     href="{% url 'inforequests:index' %}"
   {% endif %} 
>
```

Ale v tomto pripade, sa to da zapisat ovela jednoduchsie bez `if`/`else`:
```
{% url user.is_authenticated|yesno:'inforequests:mine,inforequests:index' %}
```

Vid: https://docs.djangoproject.com/en/1.7/ref/templates/builtins/#yesno

In general odporucam si vzdy, ked robis nieco komplikovanejsie, pozriet dokumentaciu, ci na to nie je nejaky tool.